Formula/gambit-scheme.rb

```diff
@@ -27,27 +26,28 @@ class GambitScheme < Formula
   def install
     args = %W[
       --prefix=#{prefix}
+      --infodir=#{info}
       --enable-single-host
-      --enable-multiple-versions
       --enable-default-runtime-options=f8,-8,t8
       --enable-openssl
     ]
 
     system "./configure", *args
 
-    # Fixed in gambit HEAD, but they haven't cut a release
-    inreplace "config.status" do |s|
-      s.gsub! %r{/usr/local/opt/openssl(?!@1\.1)}, "/usr/local/opt/openssl@1.1"
-    end
-    system "./config.status"
-
     system "make"
     ENV.deparallelize
     system "make", "install"
+
+    # fix lisp file install location
+    mkdir_p share/"emacs/site-lisp/#{name}"
+    elisp.install share/"emacs/site-lisp/gambit.el"
+
+    # some formulae may expect gambit-scheme to be internally-versioned
+    prefix.install_symlink "." => "current"
```

Comment:
It would be nice to avoid being able to recurse infinitely into the `current` subdirectory, which I think this allows.

Reply:
I guess I could remove it altogether, since `gerbil-scheme` is the only core dependent at this time, and I'm redoing the `gambit-scheme` path config there anyway.